ariadne/make_executable_schema.py

```diff
@@ -1,54 +1,14 @@
-from graphql import GraphQLSchema, parse
-from graphql.language.ast import (
-    Document,
-    ObjectTypeDefinition,
-    OperationTypeDefinition,
-    Name,
-    NamedType,
-    SchemaDefinition,
-)
-from graphql.utils.build_ast_schema import build_ast_schema
-
-from .add_resolve_functions_to_schema import add_resolve_functions_to_schema
-
-
-def build_default_schema(document: Document) -> SchemaDefinition:
-    defined_types = [
-        td.name.value
-        for td in document.definitions
-        if isinstance(td, ObjectTypeDefinition)
-    ]
-    operations = []
-    if "Query" in defined_types:
-        operations.append(
-            OperationTypeDefinition("query", type=NamedType(name=Name("Query")))
-        )
-    if "Mutation" in defined_types:
-        operations.append(
-            OperationTypeDefinition("mutation", type=NamedType(name=Name("Mutation")))
-        )
-    if "Subscription" in defined_types:
-        operations.append(
-            OperationTypeDefinition(
-                "subscription", type=NamedType(name=Name("Subscription"))
-            )
-        )
-    return SchemaDefinition(operation_types=operations, directives=[])
+from typing import List, Union
 
+from graphql import GraphQLSchema
 
-def document_has_schema(document: Document) -> bool:
-    return any(isinstance(td, SchemaDefinition) for td in document.definitions)
-
-
-def build_schema(type_defs: str) -> GraphQLSchema:
-    document = parse(type_defs)
-    if not document_has_schema(document):
-        schema_definition = build_default_schema(document)
-        document.definitions.append(schema_definition)
-    return build_ast_schema(document)
+from .add_resolve_functions_to_schema import add_resolve_functions_to_schema
+from .build_schema_from_type_definitions import build_schema_from_type_definitions
```

Comment:
These names are enormous :/

Reply:
I would try to keep module names smaller even if you prefer to match the function names of Apollo.

Reply:
@patrys I've moved some code around, made our modules area based (eg. resolvers, build schema, executable schema) and renamed modules to reflect that, giving us shorter modules now:

`resolvers.py`
`build_sechma.py`
`make_executable_schema`